style.css

```diff
@@ -74,20 +85,240 @@ p {
 
 .connect {
   margin: 12px 24px 12px 24px;
-  font-family: 'Poppins', sans-serif;
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
 }
 
-.media-icon {
+.media_icon {
   display: flex;
   margin: 24px 0 24px;
   padding-left: 24px;
 }
 
-.media-icon-item {
+.media_icon_item {
+  margin-right: 12px;
+}
+
+.work {
+  display: grid;
+  grid-template-rows: (4, 1fr);
+  margin: 114px 24px 114px 24px;
+}
+
+.project {
+  padding: 16px;
+  background-color: #fff;
+  border-radius: 16px;
+}
+
+.project_1,
+.project_2,
+.project_3 {
+  margin-bottom: 88px;
+}

```

Comment:
- [ ] It is a requirement that the content of each project is aligned with the flexbox. Kindly implement this.